Mathlib/RingTheory/Ideal/Operations.lean

```diff
@@ -1174,6 +1174,10 @@ theorem IsPrime.multiset_prod_map_le {s : Multiset ι} (f : ι → Ideal R) {P :
   simp_rw [hp.multiset_prod_le, Multiset.mem_map, exists_exists_and_eq_and]
 #align ideal.is_prime.multiset_prod_map_le Ideal.IsPrime.multiset_prod_map_le
 
+theorem IsPrime.multiset_prod_mem_iff {I : Ideal R} (hI : I.IsPrime) (s : Multiset R) :
+    s.prod ∈ I ↔ ∃ p ∈ s, p ∈ I := by
+  simpa [← span_singleton_le_iff_mem] using (hI.multiset_prod_map_le (fun r ↦ span {r}))
```

Comment:
```suggestion
  simpa [span_singleton_le_iff_mem] using (hI.multiset_prod_map_le (fun r ↦ span {r}))
```
also works. I think `span_singleton_le_iff_mem` can be a good `simp` lemma.

Reply:
Thank you for your suggestion!